modules/ROOT/pages/features/UserDefinedFunctions.adoc

```diff
@@ -98,9 +98,41 @@ As an input it receives a SIP3 packet with generic payload represented as a `Map
 
 As mentioned in the previous section, the main purpose of any SIP3 UDF is assigning `String` or `Boolean` attribute to a SIP3 packet according to its business logic.
 
-We call such attributes User-Defined Attributes (UDAs). UDAs can bring your SIP3 experience to a next level if you follow the following rule:
+We call such attributes User-Defined Attributes (UDAs). Each UDA by default is available in Search query and as Tag in metrics.
 
-IMPORTANT: Always choose `String` attributes with a limited amount of possible values. Otherwise, it may affect your database performance and can even bring it down. For instance, if you have a mobile application: `operation_system` or `application_version` are good choice of UDAs.
+But in some cases write UDA as Tag in metrics will be enough. To configure how UDA will be processed there are 3 processing mode:
+
+- _Database_ - with this mode attribute will be saved in database and available in SIP3 UI Search query.
```

Comment:
in this mode your attribute will be saved to database and provisioned as an `Advanced Search` expression